### ISSUE TITLE
Automatic metadata fetching

### ISSUE THREAD

**Issue by kirillt** [ARK-Builders/ARK-Shelf-Desktop#5]
Would be cool to automatically fetch title and description fields, as well as preview image.
Similar to how [the Android app](https://github.com/ARK-Builders/ARK-Shelf) is doing this.

**Comment by kirillt** [comment]
No preview file?
```
$ unzip link.zip 
Archive:  link.zip
 extracting: link.json
```

No description:
```
$ cat link.json | jq
{
  "title": "Solana TL;DR",
  "desc": "",
  "url": "https://2501babe.github.io/posts/solana101.html?accessToken=eyJhbGciOiJIUzI1NiIsImtpZCI6ImRlZmF1bHQiLCJ0eXAiOiJKV1QifQ.eyJhdWQiOiJhY2Nlc3NfcmVzb3VyY2UiLCJleHAiOjE2NDQ0ODUzOTUsImciOiI4QXcyeDJNQTJrUVVYdnYwIiwiaWF0IjoxNjQ0NDg1MDk1LCJ1c2VySWQiOi0xNzQ4NDU1ODc5fQ.poTnBB48mQjZhiun-HoSCPNG5pXLBoup42GxUMuSiMI",
  "created_time": {
    "secs_since_epoch": 1656745407,
    "nanos_since_epoch": 598406590
  }
}
```

**Comment by kirillt** [comment]
Solved in #11
